src/main/windows.ts

```diff
@@ -24,6 +24,15 @@ export function mainIsReady() {
   mainIsReadyResolver();
 }
 
+function isWebProtocol(url: string) {
```

Comment:
non-blocking, but we could also do something like `safelyOpenWebURL`:

```js
function safelyOpenWebURL(url: string) {
   try {
      const { protocol } = new URL(url);
      if (['http:', 'https:'].includes(protocol)) {
        shell.openExternal(details.url);
      }
   } catch {}
}
```

where we could then call e.g.:

```js
safelyOpenWebURL(details.url);
```

instead of 

```js
if (isWebProtocol(details.url)) {
   shell.openExternal(details.url);
}
```

at the callsites?

Reply:
Good suggestion! Implemented, and also added test coverage which I should have added from the start. 😅